### DISCUSSION TITLE
What settings are required to train Pony loras?

### DISCUSSION THREAD

**Discussion by Deejay85** [bmaltais/kohya_ss#3085] [Q&A]
I've trained a pony lora before, and unless I accidentally changed a setting I shouldn't have, the only option I have checked underneath the models tab is SDXL. I am, of course, using the original Pony model, it's just when the lora is done, it doesn't do anything. As for the settings here is the relevant information below along with an attachment of my .json file:

[Pony Model.json](https://github.com/user-attachments/files/18807480/Pony.Model.json)

Images: 103
Epochs: 40
Batch size: 2
LR scheduler: Cosine with restarts
Learning rate: 0.0003
Text Encoder learning rate: 0.0003
Unet learning rate: 0.0003

**Comment by mykeehu** [comment]
If you want to train a person, here is my json file. I have trained many persons with this for my own use, but the following conditions must be met:
- 100 images (so 1x100, 2x50, 4x25, 5x20, 10x10)
- I use 1024x1024 resolution for the sample images
- For low quality images you need at least 20 images, otherwise it will take over the low quality!
- In the end, a model is saved because the last one is the best.
- It needs 24 GB VRAM, otherwise you have to turn on optimizations
- have your own reg images and set the base model. The ponyRealism_v21VAE model is the best because it has been mixed into many other models.
- in the sample prompt replace xxxxx with your own token

[PonyXL1-realistic-person-Prodigy-perfect-sfw.json](https://github.com/user-attachments/files/18846036/PonyXL1-realistic-person-Prodigy-perfect-sfw.json)
